x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table.tsx

```diff
@@ -124,12 +136,36 @@ const getColumns = ({
     field: 'updatedAt',
     name: TEXT.UPDATED_AT,
     width: '15%',
-    render: timestampRenderer,
+    render: (timestamp) => moment.duration(moment().diff(timestamp)).humanize(),
```

Comment:
this should be exported to a shared function so if we need to humanize time in another table or anywhere else basically, we should use the same pattern and prevent different date patterns

Reply:
btw `moment(timestamp).fromNow()` might result the same outcome